### DISCUSSION TITLE
SSD1963 - ESP32 - 8 bit Parallel - Add support for GSL1680-F touchscreen driver

### DISCUSSION THREAD

**Discussion by iw7dmh** [Bodmer/TFT_eSPI#2136] [New processor/display support or capability request]
Hello,

I configured in a very easy way TFT_eSPI (2.4.74) libs for my 5-inch display [ER-TFTM050-5](https://www.buydisplay.com/lcd-5-ssd1963-tft-module-touch-screen-display-800x480-mcu-arduino) but had a lot of problems trying to configure the GSL1680-F touchscreen driver.
After some code "patchworking" I was able to make it works with ESP32 on Arduino Framework, so I am wondering if you can add an "official support" for this TouchScreen controller. I found it is very fast and I found it can handle up to ten fingers.
If it could help I am leaving here the working code.

Best regards
Enzo Stefanazzi

[GSL1680 - F.zip](https://github.com/Bodmer/TFT_eSPI/files/9898826/GSL1680.-.F.zip)
[User_Setup.h.txt](https://github.com/Bodmer/TFT_eSPI/files/9898841/User_Setup.h.txt)

![Connections](https://user-images.githubusercontent.com/87143145/198953353-cb9d02cf-ccce-4ef1-8d85-1393f5c4fd45.png)

**Comment by Bodmer** [comment]
Thanks for this. I think the best solution is to have an independant library that supports multiple touch controllers and interface types. I will consider this further.
